base/io.jl

```diff
@@ -448,7 +456,29 @@ function readuntil(s::IO, t::AbstractString)
 end
 
 readline() = readline(STDIN)
-readline(s::IO) = readuntil(s, '\n')
+
+function readline(s::IO, chomp::Bool=false)
```

Comment:
I'd say its relatively efficient and anyway it has the same performance as before.